cmake/std/unittests/TestPullRequestLinuxDriverTest.py

```diff
@@ -73,7 +73,7 @@ def test_verifyGit_fails_with_old_version(self):
             m_check_out.return_value='git version 1.10.1'
 
             bad_git_string = 'Git version  should be 2.10 or better - Exiting!'
-            if sys.version_info.major is not 3:
+            if(sys.version_info.major != 3):
```

Comment:
Why did you move away from python style to a c style of comparison? I always struggle with this one myself.

Reply:
In Python using `is` or `is not` is an identity test and is checking that the operands are the exact same object... where `==` and `!=` are equality checks that are using the output of the operands' `__eq__` and `__cmp__` methods.  I generally think of using `is` for singletons like `None` comparisons to see if something is exactly that thing and for value checks I tend to use equality/inequality operators.

In this case, I think using `is` is fine since sys.version_info.major is an `int` ... buuuut at least under Python 3.8 you'll get a warning using `is` instead of `!=`.  For example:
```
>>> sys.version_info.major is not 3
<stdin>:1: SyntaxWarning: "is not" with a literal. Did you mean "!="?
False
>>> sys.version_info.major != 3
False
>>>
```

I added the change to eliminate the warning.